<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lambila</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="horizontalSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="verticalSplitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="fileListLabel">
           <property name="text">
            <string>File list</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTreeWidget" name="fileTreeWidget">
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="fileButtonsLayout">
           <item>
            <widget class="QPushButton" name="refreshButton">
             <property name="text">
              <string>Refresh</string>
             </property>
             <property name="icon">
              <iconset theme="list-add" resource="resources.qrc">
               <normaloff>:/resources/icons/list-add.svg</normaloff>:/resources/icons/list-add.svg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="fileAddButton">
             <property name="text">
              <string>Add file</string>
             </property>
             <property name="icon">
              <iconset theme="list-add" resource="resources.qrc">
               <normaloff>:/resources/icons/list-add.svg</normaloff>:/resources/icons/list-add.svg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="fileRemoveButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Remove file</string>
             </property>
             <property name="icon">
              <iconset theme="list-remove" resource="resources.qrc">
               <normaloff>:/resources/icons/list-remove.svg</normaloff>:/resources/icons/list-remove.svg</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="hierarchyLayout">
         <item>
          <widget class="QLabel" name="hierarchyLabel">
           <property name="text">
            <string>Hierarchy</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTreeWidget" name="hierarchyTreeWidget">
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <layout class="QVBoxLayout" name="rightLayout">
        <item>
         <widget class="QTextEdit" name="textEdit"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="application-exit"/>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset theme="document-new" resource="resources.qrc">
     <normaloff>:/resources/icons/document-new.svg</normaloff>:/resources/icons/document-new.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;New...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="document-open" resource="resources.qrc">
     <normaloff>:/resources/icons/document-open.svg</normaloff>:/resources/icons/document-open.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="document-save" resource="resources.qrc">
     <normaloff>:/resources/icons/document-save.svg</normaloff>:/resources/icons/document-save.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset theme="document-save-as" resource="resources.qrc">
     <normaloff>:/resources/icons/document-save-as.svg</normaloff>:/resources/icons/document-save-as.svg</iconset>
   </property>
   <property name="text">
    <string>Save &amp;as...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
